tools/isobuild/bundler.js

```diff
@@ -1522,11 +1522,17 @@ class ClientTarget extends Target {
     if (this.arch === 'web.cordova') {
       const { WebAppHashing } =
         loadIsopacket('cordova-support')['webapp-hashing'];
+
       const cordovaCompatibilityVersions =
         _.object(_.map(CORDOVA_PLATFORM_VERSIONS, (version, platform) => {
-          const hash = WebAppHashing.calculateCordovaCompatibilityHash(
-            version,
-            this.cordovaDependencies);
+          let cordovaDependencies = Object.assign({}, this.cordovaDependencies);
+          let pluginsExcludedFromCompatibilityHash = process.env.CORDOVA_COMPATIBILITY_VERSION_EXCLUDE || '';
+          cordovaDependencies = _.omit(cordovaDependencies, pluginsExcludedFromCompatibilityHash.split(';'));
```

Comment:
Also, was there a particular reason for using a semi-colon (`;`) instead of a comma (`,`) as the delimiter when multiple packages are being excluded?

(Using a semi-colon makes it so the environment variable might have to be quoted on Unix.)